### ISSUE TITLE
pip install in python==3.11 fails : could not find a version that satisfies

### ISSUE THREAD

**Issue by aflip** [yahoojapan/NGT#149]
When I install with pip or pip3 inside a conda evironment that is python3.11 i get this error:

```
ERROR: Could not find a version that satisfies the requirement ngt (from versions: none)
ERROR: No matching distribution found for ngt
```
Works fine in python 3.10 environments

**Comment by masajiro** [comment]
NGT for Python 3.11 has not been released yet. We plan to release it in the near future.

**Comment by aflip** [comment]
Thank you. Is there some way I can contribute?

**Comment by masajiro** [comment]
Thank you for your offer. However, I cannot come up with a good way for you to contribute. Anyway, we will do it ASAP.

**Comment by masajiro** [comment]
Thank you for waiting. I have released ngt pypi package [v2.1.3](https://pypi.org/project/ngt/2.1.3/) for python v3.6-3.12 on Linux.
